src/borg/testsuite/archiver/create_cmd.py

```diff
@@ -780,9 +781,19 @@ def extract_hashing_time(borg_create_output):
             hashing_time = float(hashing_time.removesuffix(" seconds"))
             return hashing_time
 
+        def create_test_file():
+            # create_regular_file is not used because the test
+            # needs random data in the file to avoid deduplication
+            data = randbytes(50000)
+            filename = os.path.join(self.input_path, "test_file")
+            with open(filename, "wb") as file:
+                for byte in data:
+                    file.write(bytes(byte))
+
         # Test case set up: create a repository and a file
         self.cmd(f"--repo={self.repository_location}", "rcreate", "--encryption=none")
-        self.create_regular_file("testfile", size=6000000)
+        create_test_file()
+        # self.create_regular_file("testfile", size=3000000, contents=create_random_bytes())
```

Comment:
please do not reinvent the wheel.

you can call `self.create_regular_file("testfile", content=...)`.

Reply:
I didn't notice.
Turns out there is a bug in my `create_test_file` function, it wasn't creating a 50KB file, it actually was much longer than that because `bytes()` create an array of zeroes when you pass it an int (what I am doing in line 791). So, I was actually creating a file filled with zeroes.
Now, I modified the code to use `create_regular_file` as you said, but it now requires 6MB of random bytes to pass the test.